projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
+
+
+build-vdk-jupyterlab-extension:
+  stage: build
+  image: "python:3.8-alpine"
+  before_script:
+    - cd projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension || exit 1
+    - pip install -U pip setuptools pre-commit
+    - apk --no-cache add npm
+    - apk add rust cargo
+    - apk add libffi-dev
+    - pip install pytest-cov
+  script:
```

Comment:
Is these steps ppl might want to do locally? 
if it is then it might be best to do it in a script.

Reply:
@murphp15  in macOS these are not required and when building locally I had no problems with those